.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Enable Ruff G004 to enforce logging best practices
+4c3043200ca15ade613fa3e3f8ca84bbe3522f21
```

Comment:
What's this file for?

Reply:
When we make formatting/style changes in the code that are not useful for the `git blame`, we can ignore the commits (revisions) in which the changes were made in the relevant files to keep their history clean.

More on this here: https://graphite.dev/guides/git-blame-ignore-revs

https://github.com/orgs/community/discussions/5033

Reply:
Now that I fixed it – if you check any of the files, say, [`main`/pyodide_build/buildall.py (blame)](https://github.com/pyodide/pyodide-build/blame/main/pyodide_build/buildall.py), it wouldn't show the formatting change in the history (by default).